code/modules/events/destructible_terrain.dm

```diff
@@ -0,0 +1,13 @@
+//Toggles the destructible terrain on the ground level
+/proc/toggle_destructible_terrain()
+	var/list/turfs = get_all_of_type(/turf/closed/wall, TRUE)
+	for(var/turf/closed/wall/turf_for_editing in turfs)
+		if(!is_ground_level(turf_for_editing.z))
+			//We only want to edit ground turfs!
+			continue
+		//If the initial wall was indestructible, we toggle indestructability
+		if(initial(turf_for_editing.hull))
+			if(istype(turf_for_editing, /turf/closed/shuttle))
```

Comment:
/turf/closed/shuttle won't be getting picked up here as it's not a /turf/closed/wall